### PR TITLE
Added home page and search bar

### PR DESCRIPTION
**Pull request by MaxShapiro32**
PR for https://github.com/moficodes/bee-travels/issues/2

### CODE REVIEW

File: src/ui/client/src/components/Content.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+import {Container, Row, Col, Navbar, NavbarBrand} from 'reactstrap';
+import ReactMapboxGl from 'react-mapbox-gl';
+import Autosuggest from 'react-autosuggest';
 
 const API_BASE_URL = "http://localhost:4000"; //process.env.API_URL;
 
 const Map = ReactMapboxGl({
     accessToken:
-      'pk.eyJ1IjoibWFnaWNtYXgzMiIsImEiOiJjanR0MGxubXcweHVzNDVwaWV2MGQ1ZWp3In0.8YiTJ8uuaq831eLcsjc03w'
+      'pk.eyJ1IjoibWFnaWNtYXgzMiIsImEiOiJjanR0MGxubXcweHVzNDVwaWV2MGQ1ZWp3In0.8YiTJ8uuaq831eLcsjc03w' //process.env.Mapbox_Access_Token;
```
Comment (moficodes) [inline src/ui/client/src/components/Content.js:10]: Should we think of a way to remove the access tokens from our source?

**Review by moficodes — APPROVED** [review]
Lets think about changing our mapbox api and removing it from github.

File: src/ui/client/src/components/Content.js (hunk 1 above)
Comment (MaxShapiro32) [reply src/ui/client/src/components/Content.js:10]: Yes. I shouldn't have had it in there. We can incorporate it as an environment variable (which is why I added that comment)

Comment (moficodes) [reply src/ui/client/src/components/Content.js:10]: Already merged. just to make sure it stays fresh in our mind.

**Pull request merged**